consistency-tests/__tests__/clickables.test.js

```diff
@@ -35,15 +35,15 @@ const IconButtonWrapper = (props: any) => (
 );
 
 describe.each`
-    Component            | name
-    ${ActionItem}        | ${"ActionItem"}
-    ${Button}            | ${"Button"}
-    ${ClickableWrapper}  | ${"Clickable"}
-    ${CompactCell}       | ${"CompactCell"}
-    ${DetailCell}        | ${"DetailCell"}
-    ${IconButtonWrapper} | ${"IconButton"}
-    ${Link}              | ${"Link"}
-`("$name with an href", ({Component, name}) => {
+    Component            | name             | role
+    ${ActionItem}        | ${"ActionItem"}  | ${"menuitem"}
+    ${Button}            | ${"Button"}      | ${"button"}
+    ${ClickableWrapper}  | ${"Clickable"}   | ${"link"}
+    ${CompactCell}       | ${"CompactCell"} | ${"link"}
+    ${DetailCell}        | ${"DetailCell"}  | ${"link"}
+    ${IconButtonWrapper} | ${"IconButton"}  | ${"link"}
```

Comment:
**question:** why do `Button` and `IconButton` have different roles?

Reply:
I think we probably want it to have `role="link"` since it behave like one even if it doesn't look like one.  I'm kind of shaky with my a11y knowledge so you may want to get a second opinion.